DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/RefreshUIDProcessor.cs

```diff
@@ -0,0 +1,51 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class RefreshUIDProcessor : IAnonymizerProcessor
+    {
+        public Dictionary<string, string> ReplacedUIDs { get; } = new Dictionary<string, string>();
```

Comment:
Looks like the ReplaceedUIDs serves as a cache of created UIDs.
But the Processor object is created for each rule, and the Process method is called once for one item.
So, as an instance variable (instead of static class variable), the dictionary is always empty when Process method is called.
If the purpose of dictionary is truly like this, change it to static variable.

Reply:
Fixed it.